### DISCUSSION TITLE
App restarts to splash screen while, during streaming, I put in background

### DISCUSSION THREAD

**Discussion by al3x88** [HaishinKit/HaishinKit.swift#1628] [Q&A]
Hi, after updated to latest version (2.0.0) I noticed that sometimes the app restarts to splash screen when is placed in background.
It happened just after 10 seconds or something being in background....
Is there a way to avoid this? 

Any configuration is needed in [applicationDidEnterBackground] method?

thanks so much!

**Comment by shogo4405** [comment]
I wasn’t aware of that behavior in the background. I think it might be a bug, so I’ll look into it.

**Reply by al3x88** [reply]
Thanks!! The issue is related to MediaMixer().
Here you find my extensive troubleshooting for the issue https://github.com/shogo4405/HaishinKit.swift/issues/1629

**Comment by shogo4405** [comment]
close fixed https://github.com/shogo4405/HaishinKit.swift/issues/1629
